Source/settings/mods.ts

```diff
@@ -49,6 +49,37 @@ module FullScreenPokemon {
                         stats.speed = this.settings.objects.properties.Player.speed;
                     }
                 }
+            },
+            {
+                name: "Level 100",
+                enabled: false,
+                events: {
```

Comment:
What happens when Pokemon are added or removed to/from the party? You should handle that case as well.